1.0.0-beta4/Dockerfile

```diff
@@ -0,0 +1,37 @@
+FROM mono:3.12
```

Comment:
Mono doesn't compile without a mono base image. Actually kind a strange. I have changed the base image to `mono:4`